### DISCUSSION TITLE
Sync paused

### DISCUSSION THREAD

**Discussion by iMongui** [iterate-ch/cyberduck#16346] [Q&A] [answered]
Today we bought a license to synchronise (because we were using free version for a while and we thought that it deserves the money), but for any weird reason, the sync process of a Google Drive account is stuck on sync paused once you click on resume sync. It starts to sync but after 1 minute or so, it pauses again the process and you need to force a resume over and over and over again, which is very exhausted to follow up. The config is using smart synchro to be able to have offline version of the files

OS: Windows Server 2019
Data source: Google Drive licensed corporative account, around 200GB of data used
Firewall open
Network ok, no issues
Enough disk space on destination, 500GB of free data

How its supposed that synchronization can work on auto mode and avoid this sync paused state?

**Comment by tatze96** [comment, marked as answer]
Some general information regarding paused synchronization:
The synchronization can be paused manually but also gets paused by Mountain Duck if the connection to the server is not possible, is lost or is too unstable to transfer files. Files and folders that weren't indexed before the server or network connection dropped won't show up if you work offline.

In the case of Mountain Duck pausing the synchronization process due to connectivity issues, Mountain Duck performs reachability checks at regular intervals and will resume the synchronization automatically once the connection can be established again.

If the paused state is still present, [enable debug logging](https://docs.cyberduck.io/mountainduck/support/#debug-log), recreate the issue and send us the log files with the ticket reference GH-16139 to support@mountainduck.io for further investigation.

**Comment by nrktkt** [comment]
I see this is answered, but is there any way to work around it? 
My use case is for photos; if I dump a shoot's worth of photos at once and walk away from my computer, I'd like them to be uploaded when I come back, even if there is some intermittent connection issue during that time.
